Create list of labels to use in the sdk's issues (#202)
meilisearch/integration-guides: guides/sdk-labels.md

# SDK's issues Labels

⚠️ These labels are not the same as this repo's labels. They are meant to be used to tag the issues opened in the different SDK's repositories.

We have many issues in every integration repository. These help us understand what each issue is about or its state.
A clear label prevents us from re-reading the issue when we are scanning them, and they provide a context in the blink of an eye.

Nonetheless, until now, every repo used its own label system. Thus, to avoid adapting to every repo's label choices, we want to create a consistent labels list to use.

These are the labels that are encouraged to use.

> ## Default labels

Reviewer: I don't know if we should create two different sections @bidoubiwa they all have the same weight even if they were created by Github or by us.

Author: Okey! I hesitated so I don't have a real 100% opinion on the matter

They already exist in every repo.

| label Name  | Color  | description  |
|---|---|---|
|`bug` | #d73a4a | Something isn't working |
| `good first issue` | #7057ff | Good for newcomers |
| `help wanted` | #008672 | Extra attention is needed |
| `enhancement` | #a2eeef | New feature or request |

> | `security` | #ee0701 | Pull request or issue that has recieve no activity for a long time. |

Reviewer: ```suggestion
| `security` | #ee0701 | Pull request or issue that address some security concern. |
```

Author: Oh thanks for spotting this! 
if we want to stay in line with [these labels](https://github.com/meilisearch/meilisearch/labels) it should be described this way: 

> Address a security vulnerability

What do you think?

Reviewer: Good idea!

## Custom labels

They need to be added manually

| label Name  | Color  | description |
|---|---|---|

> | `support` | #F78FB2| Issues related to support questions. |

Reviewer: Maybe we can add that they will be handled by the dev rel team?

Reviewer: I think we don't need to be in line with them, if you find it a good idea you can add it, no hard feelings :D
